.circleci/config.yml

```diff
@@ -12,17 +12,22 @@ jobs:
     steps: 
       - checkout
       - run:
-          name: Download CLI
+          name: "Create SFDX Folder" 
           command: |
             # Create sfdx directory
-            mkdir sfdx
-            # Download Salesforce CLI
-            wget -qO- $DX_CLI_URL | tar xJ -C sfdx --strip-components 1
+            mkdir ~/sfdx
+      - run:
+          name: Download CLI
+          command: |
+            # Download cli
+            # By default, the script installs the current version of Salesforce CLI. To install the release candidate, set the DX_CLI_URL_CUSTOM local variable to the appropriate URL
```

Comment:
up in line 9, that needs to be https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-amd64.tar.xz (new urls) and change those comment urls to point people here: https://developer.salesforce.com/tools/sfdxcli

Reply:
line 5...probably update to LTS.  Circle lets you say
`- image: cimg/node:lts`

to avoid ongoing maintenance from hardcoded node versions.

Reply:
line 40: let's remove that!

Reply:
line 66/67...there's sometime when 5 min is too much, or too little.  Let's remove that wait/sleep step, and use the 
```
-b, --publishwait=publishwait                                                     number of minutes to wait for subscriber package version ID to become available in the target org
```
from force:package:install to properly wait for the package replicaiton.